src/Facebook/InstantArticles/Elements/Audio.php

```diff
@@ -212,4 +217,14 @@ public function toDOMElement($document = null)
 
         return $element;
     }
+
+    /**
+     * Overrides the @see Element::isValid().
+     *
+     * @return true for valid Audio that contains not empty url, false otherwise.
+     */
+    public function isValid()
+    {
+        return !Type::isTextEmpty($this->url);
```

Comment:
Definetely this should be an issue because links can:
- have the protocol notation: //domain.com/path/image.png
- Use base64
- Use relative link

All the cases we have to resolve the final link and check if it is a valid Image.
Opened issue: #98